src/test/spec/unified_runner/test_file.rs

```diff
@@ -164,6 +168,9 @@ pub(crate) struct Client {
     #[serde(default, deserialize_with = "deserialize_server_api_test_format")]
     pub(crate) server_api: Option<ServerApi>,
     pub(crate) store_events_as_entities: Option<Vec<StoreEventsAsEntity>>,
+    #[cfg(feature = "tracing-unstable")]
+    #[serde(default, deserialize_with = "deserialize_tracing_level_map")]
+    pub observe_log_messages: Option<HashMap<String, tracing::Level>>,
```

Comment:
```suggestion
    pub(crate) observe_log_messages: Option<HashMap<String, tracing::Level>>,
```
(small nit, i think we decided upon using `pub(crate)` for internal test stuff)

Reply:
oh yeah - updated